aea/cli/add.py

```diff
@@ -41,6 +41,20 @@ def add(ctx: Context):
     _try_to_load_agent_config(ctx)
 
 
+@add.command()
+@click.argument('connection_name', type=str, required=True)
```

Comment:
Why do we distinguish here between type and name? Aren't they the same?

Reply:
I think it is because at the moment you do this differently than for the skills approach (i.e. the connections don't get copied in). I think they should though. Happy to discuss.